product/opni/components/Cluster.vue

```diff
@@ -117,26 +242,67 @@ export default {
         add-label="Add Label"
       />
     </div>
-    <Card class="m-0 mb-20" :show-highlight-border="false" :show-actions="false">
+    <Card class="m-0 mb-10" :show-highlight-border="false" :show-actions="false">
       <h4 slot="title" class="text-default-text">
         Install Command
       </h4>
-      <div slot="body" class="">
-        <CopyCode class="install-command">
-          helm install ..... --token={{ token }}
+      <div slot="body">
+        <div v-if="inputArgs.length > 0" class="row">
+          <div v-for="item in inputArgs" :key="item.key" class="options col span-3 mb-10">
+            <LabeledSelect
+              v-if="item.kind === 'select'"
+              v-model="item._data.value"
+              :label="item._data.label"
+              :required="item._data.required"
+              :options="item.select.items.map(item => ({
```

Comment:
It's best to keep as much code out of the templates as possible. It can really get things cluttered fast. This should either be a property in data or a computed property depending on when the data is available.